_includes/faq/auto-generate-unique-ids_v1.1.html

```diff
@@ -0,0 +1,21 @@
+To auto-generate unique row IDs, use the [`SERIAL`](serial.html) data type, which is an alias for [`INT`](int.html) with the `unique_rowid()` [function](functions-and-operators.html#id-generation-functions) as the [default value](default-value.html):
```

Comment:
Yes, I think we should update this language. UUID is not just for interoperability - it helps to spread load around the cluster and is a good choice overall. UUID is probably a better choice than SERIAL for top-level tables (those without an INTERLEAVE IN PARENT clause) unless you specifically need the time-locality that SERIAL provides.